cmd/create/cluster/cmd.go

```diff
@@ -493,13 +517,53 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	// Autoscaling
+	autoscaling := args.autoscalingEnabled
+	if !autoscaling && interactive.Enabled() {
```

Comment:
You should check for `!cmd.Flags().Changed("enable-autoscaling")` - if the user explicitly set `--enable-autoscaling=false` we probably shouldn't force the interactive question on them

Reply:
fixed